MarsRoverBestGroup3.0/MarsRoverBestGroup3.0/Utilities/MarsDate.cs

```diff
@@ -50,21 +50,65 @@ public static double MarsToEarthDate(DateTime dateRequest)
 
         }
         */
-        public static double earthConversionRate = 86400.0 / 88775.0;
+         
+        public static int earthConversionRate = 86400/ 88775;
```

Comment:
This one does need to be a double though.
(otherwise it will try to covert 0.97 to an integer and you'll just end up with 0 :))

Reply:
I'm confused, is this line 54 or 53?

Reply:
yay so are you happy for me to merge?